unittests/rem_voting_test.cpp

```diff
@@ -0,0 +1,339 @@
+/**
```

Comment:
as I can see this is modified bootseq_tests.cpp. I propose to keep the same name for test to see actual modification

Reply:
this is new test for voting functionality (I will append there more logic when it comes) as well I've update `unittests/bootseq_tests.cpp` so it will pass